src/components/attendee/Popover.tsx

```diff
@@ -6,20 +6,33 @@ import { POPOVER_CONTAINER_ID } from "settings";
 
 export interface PopoverProps {
   referenceElement?: Element | null;
+  offset?: number[];
 }
 
 export const Popover: React.FC<PopoverProps> = ({
   children,
   referenceElement,
+  offset,
 }) => {
   const popoverContainerElement = document.querySelector(
     `#${POPOVER_CONTAINER_ID}`
   );
+  const [left, top] = offset ?? [];
```

Comment:
not to create an empty object every time.
```suggestion
  const [left, top] = offset ?? ALWAYS_EMPTY_ARRAY;
```

Reply:
[a73b96b](https://github.com/sparkletown/sparkle/pull/3008/commits/a73b96be5689fae99a7a121493fbadbc06abb504)